share/examples/bitcoin.conf

```diff
@@ -1 +1 @@
-# This is a placeholder file. Please follow the instructions in `contrib/devtools/README.md` to generate a bitcoin.conf file.
+# This is a placeholder file. Please follow the instructions in the `gen-bitcoin-conf.sh` section of `contrib/devtools/README.md` to generate a bitcoin.conf file.
```

Comment:
I don't like this change that much since this hardens the logical link between this file and `contrib/devtools/README.md` without any kind of automation to check if e.g. the section `gen-bitcoin-conf.sh` still exist.

Reply:
Good point. My intention here is to reduce ambiguity by being more explicit to the user.

One solution would be to add a check to ensure this section mentioned actually exists in the markdown, but it's probably not worth the added code complexity for little benefit.

Do you prefer to revert this change back?

Reply:
Agreed. Resolved with aef43a3a996618d581e0a16f7fc5e842708e83c9.